client/go/src/utils/print.go

```diff
@@ -37,6 +38,18 @@ func Detail(messages ...string) {
     print("\033[33m", messages)
 }
 
+// Prints all the text of the given reader
+func PrintReader(reader io.ReadCloser) {
```

Comment:
Nit: Change to `io.Reader` to signal that this will never close the reader.

Reply:
better; thanks